### PR TITLE
fix: Updating error message and placeholders for cloud instance

### LINKED ISSUE #19097: [Bug]: Invite modal email placeholder text should be "enter email adddress(es)" Instead of "enter email address(es)  or groups"
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Invite modal email placeholder text should be "enter email adddress(es)" Instead of "enter email address(es)  or groups"
image

### Steps To Reproduce

1. login as admin
2.click on share button 
3.The placeholder text displayed on the email placeholder is wrong

### Public Sample App

_No response_

### Issue video log

_No response_

### Version

release

### PR DESCRIPTION
**Pull request by ankitakinger**
## Description

> Updating error message and placeholders for cloud instance

Fixes #19097 #19096 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

- Manual
- Jest
- Cypress

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/src/pages/workspace/settings.tsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ export default function Settings() {
 
   useEffect(() => {
     if (currentWorkspace) {
-      setPageTitle(`Members in ${currentWorkspace?.name}`);
+      setPageTitle(`${currentWorkspace?.name}`);
```
Comment (berzerkeer) [inline app/client/src/pages/workspace/settings.tsx:109]: Are we going to always show only the workspace name alone ? I thought in `Members` page we should still show, `Members in <workspaceName>` and only in `Settings `page we show just the workspace name.

**Review by berzerkeer — APPROVED** [review]

File: app/client/src/pages/workspace/settings.tsx (hunk 1 above)
Comment (ankitakinger) [reply app/client/src/pages/workspace/settings.tsx:109]: Yes, only the workspace name no matter which is the active tab.

**Review by Parthvi12 — DISMISSED** [review]

**Review by berzerkeer — APPROVED** [review]

**Pull request merged**